internal/website/content/howto/blob/open-bucket.md

```diff
@@ -26,6 +26,18 @@ both forms for each storage provider.
 ["blank import"]: https://golang.org/doc/effective_go.html#blank_import
 [documentation on URLs]: {{< ref "/concepts/urls.md" >}}
 
+## Prefixed Buckets {#prefix}
```

Comment:
Optional: This seems like more of an advanced topic, consider moving to the end.

Reply:
I'll make it a h3. I don't think anyone will notice it at the end after all of the provider-specific stuff.